staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -115,6 +118,10 @@ func (o AggregatorOptions) Validate(args []string) error {
 
 // Complete fills in missing Options.
 func (o *AggregatorOptions) Complete() error {
+	// set feature gate emulation version before Validate().
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.EmulationVersion) {
+		return utilfeature.DefaultMutableVersionedFeatureGate.SetEmulationVersion(serverversion.Effective.EmulationVersion())
```

Comment:
Similar comment to one in apiextensions-apiserver. I find it a little odd aggregator is thinking about this despite not managing flags for it

Reply:
It has the flags through `ServerRunOptions.AddUniversalFlags`

Reply:
ack. continuing discussion in the other comment. can mark as resolved